internal/commands/result.go

```diff
@@ -807,8 +816,8 @@ func printPoliciesSummary(summary *wrappers.ResultSummary) {
 			}
 			fmt.Printf("\n")
 		}
+		fmt.Printf("\n")
```

Comment:
if there are no violation, this "\n" will not be printed, is this correct?
Before your changed, we always printed it

Reply:
Yes, because before my changes always we printed the header, and now we didn't print anything.